config.toml

```
@@ -300,15 +300,15 @@ languageName ="English"
 weight = 1
 languagedirection = "ltr"
 
-[languages.zh]
+[languages.zh-cn]
 title = "Kubernetes"
 description = "生产级别的容器编排系统"
 languageName = "中文 Chinese"
 weight = 2
 contentDir = "content/zh"
```

The links (like [here](https://deploy-preview-33050--kubernetes-io-main-staging.netlify.app/zh-cn/docs/concepts/overview/what-is-kubernetes/#what-s-next) and [there](https://deploy-preview-33050--kubernetes-io-main-staging.netlify.app/zh-cn/docs/concepts/overview/components/#control-plane-components)) are still pointing `/zh/`, and I guess that this is because:

```suggestion
contentDir = "content/zh-cn"
```

But it might be okay, since the PR #33051 is already covering this.

Previews of the PR #33051 looks fine:
- https://deploy-preview-33051--kubernetes-io-main-staging.netlify.app/zh-cn/docs/concepts/overview/what-is-kubernetes/#what-s-next
- https://deploy-preview-33051--kubernetes-io-main-staging.netlify.app/zh-cn/docs/concepts/overview/components/#control-plane-components

---

Cannot change it yet. Because we don't have the `content/zh-cn` directory yet in this PR.